### PR TITLE
my first branch commit

### PR DESCRIPTION
**Pull request by kchia**
please review my commit and approve it!!!!!

### CODE REVIEW

File: .c9/project.settings [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
+    "build": {
```
Comment (kchia) [inline .c9/project.settings:2]: this is wrong!

Comment (angelicahill) [reply .c9/project.settings:2]: Not sure about this ;( have you tried GO?

**Review by jessicanorthrop — COMMENTED** [review]
I don't know what I am looking at ;)

File: .c9/project.settings (hunk 1 above)
Comment (loidav) [reply .c9/project.settings:2]: I think it is really cool. @kchia

**Review by lezahome — APPROVED** [review]
Please refactor in go. And also use the UI more

**Review by angelicahill — CHANGES_REQUESTED** [review]
Change to Go!!

File: .c9/metadata/tab1 [hunk 2]
```diff
@@ -1 +1 @@
-{"filter":false,"title":"bash - \"ip-172-31-22-116\" ","tooltip":"bash - \"ip-172-31-22-116\" ","undoManager":{"mark":-1,"position":-1,"stack":[]},"terminal":{"id":"cloud9_terminal_615","cwd":"","width":662,"height":446,"scrollTop":269},"timestamp":1548901160502,"hash":"da39a3ee5e6b4b0d3255bfef95601890afd80709"}
\ No newline at end of file
+{"filter":false,"title":"bash - \"ip-172-31-22-116\" ","tooltip":"bash - \"ip-172-31-22-116\" ","undoManager":{"mark":-1,"position":-1,"stack":[]},"terminal":{"id":"cloud9_terminal_28","cwd":"","width":653,"height":752,"scrollTop":2159},"timestamp":1549500843469,"hash":"da39a3ee5e6b4b0d3255bfef95601890afd80709"}
```
Comment (vnessawithaneye) [inline .c9/metadata/tab1:1]: wrong width

**Pull request merged**